website/components/Footer.tsx

```diff
@@ -145,21 +145,19 @@ function SubmitForm() {
   const [email, setEmail] = useState("");
   const router = useRouter();
 
-  const [message, setMessage] = useState("");
-
   const subscribe = async (e: FormEvent) => {
     e.preventDefault();
-    // 3. Send a request to our API with the user's email address.
-    const res = await fetch(`/api/subscribe?email=${encodeURI(email)}`);
-
-    const { error } = await res.json();
 
-    if (error) {
-      // 4. If there was an error, update the message in state.
-      setMessage(error);
+    const params = new URLSearchParams();
+    params.append("EMAIL", email);
+    params.append("u", "5a433922a8db867a38c8a4a8e");
+    params.append("id", "4a9aca3509");
+    params.append("c", "?");
 
-      return;
-    }
+    await fetch(
+      `https://cloudquery.us1.list-manage.com/subscribe/post-json?${params.toString()}`,
```

Comment:
We need `post-json` and `no-cors` due to a Mailchimp limitation, which means we can't get the text response and check for errors.
If we want to handle errors we need to use https://github.com/camsong/fetch-jsonp to handle this `jsonp` request.
I don't expect this to fail so I think that's OK (it only failed for me when I tried to submit too many times due to rate limiting on Mailchimp side)